README.md

```diff
@@ -1,9 +1,33 @@
+<div align="center">
+
+<img src="https://raw.githubusercontent.com/Sol-Client/client/develop/src/main/resources/assets/sol_client/textures/gui/icon.png" alt="Sol Client's logo">
+
 # Sol Client
-A nice Minecraft (non-hacked) client for 1.8.9 (and possibly newer versions in future - for an alternative [AxolotlClient](https://axolotlclient.github.io) is available)!
 
-<a href="https://discord.gg/TSAkhgXNbK"><img alt="Join Discord" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/social/discord-plural_vector.svg"/></a>
+An **open source**, non-hacked client for Minecraft 1.8.9 (newer versions coming in the future)
+
+<a href="https://www.java.com"><img alt="Java" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/built-with/java_vector.svg"></a>
+<a href="https://discord.gg/TSAkhgXNbK"><img alt="Discord Server" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/social/discord-plural_vector.svg"></a>
+<a href="https://sol-client.github.io"><img alt="Website" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/documentation/website_vector.svg"></a>
+
+</div>
+
+## 🤔 Why Sol over others?
+
+**→ Easy to use** <br />
+Everything you want to have is where want it to be. No confusing options, insatllation etc. It _just works_
+
+**→ Consistent** <br />
+One menu, one UI. Everything's listed the same way.
+
+**→ Open source and non-profit** <br />
+Anyone can modify, redistribute or see the source code of Sol. If you have privacy concerns or curiosity, you can also see for yourself that you are _not_ being spied on.
+
+**→ Active development** <br />
+_Every single_ suggested feature is considered and most likely added or planned to be added.
```

Comment:
:P I don't think we can make this promise lol.

Reply:
Sorry

Reply:
I tweaked it